test/run-spec-wasm2c.py

```diff
@@ -665,7 +665,9 @@ def main(args):
 
             # Run the resulting binary
             if options.run:
-                return subprocess.run([main_exe]).returncode
+                test_runner = os.getenv('WASM2C_TEST_RUNNER')
```

Comment:
I think can just write this in one line with  `command = os.getenv('WASM2C_TEST_RUNNER', '').split()`

Reply:
@sbc100 Fixed!